src/stream_management/mod_stream_management_mnesia.erl

```diff
@@ -140,10 +135,7 @@ start_cleaner(Opts) ->
 start_link(Opts) ->
     gen_server:start_link({local, stream_management_stale_h}, ?MODULE, [Opts], []).
 
-init([Opts]) ->
-    %% In seconds
-    RepeatAfter = proplists:get_value(stale_h_repeat_after, Opts, 1800),
-    GeriatricAge = proplists:get_value(stale_h_geriatric, Opts, 3600),
+init([#{stale_h_repeat_after := RepeatAfter, stale_h_geriatric := GeriatricAge}]) ->
```

Comment:
How about removing the `stale_h` prefixes? Also: is there a need to wrap the opts in a list?

Reply:
Hmm, probably no reason, this was the way it was here, but I'll drop the list.